deploy/sre-prometheus/ocm-agent/legacy-ingress/100-ocm-agent.PrometheusRule.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-managed-notification-alerts
+    role: alert-rules 
+  name: sre-ingresscontroller-managed-notification-alerts
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-managed-notification-alerts
+    rules:
+    - alert: MultipleIngressControllersDetectedNotificationSRE
+      expr: sum(max_over_time(splunkforwarder_audit_filter_exposed_splunk_event_total{alert="UserCreatedIngressControllerDetected"}[5m]) or vector(0)) - sum(max_over_time(splunkforwarder_audit_filter_exposed_splunk_event_total{alert="UserCreatedIngressControllerDetected"}[5m] offset 5m) or vector(0)) > 0
+      for: 30m
+      labels:
+        severity: Info
+        namespace: openshift-logging
```

Comment:
why there is a label for openshift-logging?

Reply:
@bmeng that's a great question.
I think that this was a typo in the original pull request (https://github.com/openshift/managed-cluster-config/pull/1394/files#diff-f1c65c3034f2531ee7977527eeb8c9e092a16e3ed2c6d036e39408ef23707f2dR18). I just copied the rule completely and put it behind the feature toggle, without reading what the rule actually contained. 

I've removed the namespace label from the notification. Let me know if it looks okay to you.